AidanStuff/File Converter/File Converter/Convert.cs

```diff
@@ -0,0 +1,76 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.IO;
+using System.Xml;
+
+namespace File_Converter
+{
+    public class Convert
+    {
+        public static void TextToHtml(string[] args)
```

Comment:
The Main method has to take an array of strings as its parameter because the user can type any number of arguments on the command line, but for a method like this it makes sense to declare more specific parameters that make sense for that method, e.g.,

   public static void TextToHtml(string inputFileName, string outputFileName) { ... }

This makes the method's job easier because you don't have to check the number of arguments and run time, and also makes it clearer to the caller what inputs are expected.